nemo/core/classes/modelPT.py

```diff
@@ -1032,7 +1032,7 @@ def on_validation_epoch_end(self) -> Optional[Dict[str, Dict[str, torch.Tensor]]
                 self.validation_step_outputs[dataloader_idx].clear()  # free memory
 
             if 'log' in output_dict:
-                self.log_dict(output_dict.pop('log'), on_epoch=True)
+                self.log_dict(output_dict.pop('log'), on_epoch=True, sync_dist=sync_metrics)
```

Comment:
Can I ask why L974 `self.log_dict(output_dict.pop('log'), on_epoch=True)` is not getting changed?
Shouldn't we also need to change line 974?

Reply:
yes you are correct, I didn't notice this change only covers multi-dataloader case but not single-dataloader case.